Sources/Hummingbird/Middleware/MetricsMiddleware.swift

```diff
@@ -49,17 +49,17 @@ public struct MetricsMiddleware<Context: RequestContext>: RouterMiddleware {
             // Don't record uri in 404 errors, to avoid spamming of metrics
             if let endpointPath = context.endpointPath {
                 dimensions = [
-                    ("hb_uri", endpointPath),
-                    ("hb_method", request.method.rawValue),
+                    ("uri", endpointPath),
+                    ("method", request.method.rawValue),
                 ]
-                Counter(label: "hb_requests", dimensions: dimensions).increment()
+                Counter(label: "hb.requests", dimensions: dimensions).increment()
             } else {
                 dimensions = [
-                    ("hb_uri", "NotFound"),
-                    ("hb_method", request.method.rawValue),
+                    ("uri", "NotFound"),
+                    ("method", request.method.rawValue),
                 ]
             }
-            Counter(label: "hb_errors", dimensions: dimensions).increment()
+            Counter(label: "hb.errors", dimensions: dimensions).increment()
```

Comment:
```suggestion
            Counter(label: "http.server.errors", dimensions: dimensions).increment()
```

Reply:
Not in the spec, using `hb.request.errors` as in an error caused by a request